src/transformers/models/sam/image_processing_sam.py

```diff
@@ -418,6 +430,45 @@ def post_process_masks(
 
         return output_masks
 
+    def post_process_masks_tf(
```

Comment:
Have we started including separate post-processing ops in native TensorFlow? I thought they were NumPy only. This is indeed nice.

Reply:
I wasn't sure about this - there's probably some code duplication in the processor I can remove.

Reply:
Sure! And sorry - I basically rushed through the processor code so I could get to the bit I was hype about (benchmarking GPT-4's translations)